pyaerocom/aeroval/obsentry.py

```diff
@@ -111,6 +114,8 @@ class ObsEntry(BaseModel):
     instr_vert_loc: str | None = None
     is_superobs: bool = False
     only_superobs: bool = False
+    is_bulk: bool = False
+    bulk_options: dict[str, dict] = {}
```

Comment:
Make `bulk_options` a `pydantic` struct to document the configuration options and get validation for free, something like
```python
class BulkOptions(BaseModel):
    vars: tuple[str, str]
    model_exists: ??
    mode: Literal["product", "fraction"]
    units: str

bulk_options: dict[str, BulkOptions]
```

Reply:
Added